src/psyclone/dynamo0p3.py

```diff
@@ -4305,6 +4612,10 @@ def _setup(self, ktype, module_name, args, parent):
         self._cma_operation = ktype.cma_operation
         self._fs_descriptors = FSDescriptors(ktype.func_descriptors)
 
+        # Record whether or not the kernel meta-data specifies that this
+        # is an inter-grid kernel
+        self._is_intergrid = ktype.is_intergrid
```

Comment:
Parent method needs a docstring with interface info.

Reply:
I'm confused - the parent `_setup()` method does have a docstring with interface info. Maybe this was brought in when you merged with master?

Reply:
Quite right, I'm not sure what happened there.